arch/x86/kvm/pkvm/pkvm.c

```diff
@@ -1,9 +1,11 @@
 // SPDX-License-Identifier: GPL-2.0
 #include <linux/types.h>
 #include <linux/kvm_host.h>
+#include <asm/fpu/xcr.h>
 #include "init_finalize.h"
+#include "pkvm/lapic.h"
```

Comment:
Why exactly need to change "lapic.h" to "pkvm/lapic.h"?

I guess it is related to possible ambiguity between `arch/x86/kvm/lapic.h` and `arch/x86/kvm/pkvm/lapic.h`. But the header in the current directory (i.e. in `arch/x86/kvm/pkvm/`) takes precedence anyway?

I've tried changing it back to "lapic.h" and it seems to compile without problems, with both gcc and clang.

Reply:
> So we cannot have two lines of #include "lapic.h". 

Yeah, but if "lapic.h" resolves to arch/x86/kvm/pkvm/lapic.h instead of arch/x86/kvm/lapic.h anyway, changing it to "pkvm/lapic.h" will not help, i.e. if we have both "lapic.h" and "pkvm/lapic.h" then both will just include the same file. :)

Anyway, looks like we are in agreement here.